test/eslint.config.js

```diff
@@ -8,7 +8,7 @@ import playwright from 'eslint-plugin-playwright'
 export const testEslintConfig = [
   ...rootEslintConfig,
   {
-    ignores: [...defaultESLintIgnores, '**/payload-types.ts'],
+    ignores: [...defaultESLintIgnores, '**/payload-types.ts', 'jest.setup.js'],
```

Comment:
Did you mean to include this?

Reply:
Yep - PR was failing to lint without it. We need to work on a better pattern to ensure ts-eslint only lints files it _should_ lint, but that'll be out of scope